src/stack/lib/inline-stack.scss

```diff
@@ -0,0 +1,27 @@
+.tbds-inline-stack {
+  --gap: #{$tbds-space-2};
+
+  align-items: center;
+  display: flex;
+}
+
+.tbds-inline-stack--gap-2x {
+  --gap: #{$tbds-space-3};
```

Comment:
the disconnect between the `2x` and the `tbds-space-3` value is a little weird to me. I guess it centers around where the sizing system starts (in this case, at `tbds-space-2`).

This makes me curious, can you change variables using their current value?
```css
--gap: var(--gap) * 2;
```

Reply:
Yep, I wondered if this would be confusing, so thanks for commenting on it. I think just matching the modifier name to the space value name makes it easier to comprehend.

> can you change variables using their current value?

Good question! Doesn’t seem like you can? https://codepen.io/thoughtbot/pen/oRXPLJ?editors=1100

Reply:
Resolved in 8c3489792da034972610f0a6312f432ef6bdfcae